src/app/recommendations.ts

```diff
@@ -112,4 +113,33 @@ export const RECOMMENDATIONS: Step[] = [
   { possibleIn: 1000, necessaryAsOf: 1000, level: 3, step: 'day-periods-crossing-midnight', action: 'If you use either `formatDate()` or `DatePipe` and any of the `b` or `B` format codes, the logic has been updated so that it matches times that are within a day period that spans midnight, so it will now render the correct output, such as at `night` in the case of English.' },
   { possibleIn: 1000, necessaryAsOf: 1000, level: 3, step: 'urlmatcher-null', action: 'If you use the `UrlMatcher`, the type now reflects that it could always return `null`.' },
   { possibleIn: 1000, necessaryAsOf: 1000, level: 1, step: 'v10-more-details', action: 'For more details about deprecations, automated migrations, and changes visit the [guide angular.io](https://angular.io/guide/updating-to-version-10)' },
+  { possibleIn: 1020, necessaryAsOf: 1020, level: 2, step: 'universal-baseurl', action: 'For Angular Universal users, if you use `useAbsoluteUrl` to setup `platform-server`, you now need to also specify `baseUrl`.' },
+  
+  { possibleIn: 1100, necessaryAsOf: 1100, level: 1, step: 'v11 ng update', action: 'Run `ng update @angular/core @angular/cli` which should bring you to version 11 of Angular.' },
+  { possibleIn: 1100, necessaryAsOf: 1100, level: 1, step: 'v11 versions', action: 'Angular now requires [TypeScript 4.0](https://devblogs.microsoft.com/typescript/announcing-typescript-4-0/). `ng update` will migrate you automatically.' },
+  { possibleIn: 1100, necessaryAsOf: 1100, level: 1, step: 'v11 browser support', action: 'Support for IE9, IE10, and IE mobile has been removed. This was announced in the [v10 update](http://blog.angular.io/version-10-of-angular-now-available-78960babd41#c357). ' },
+  { possibleIn: 1100, necessaryAsOf: 1100, level: 2, step: 'webpack5 optin', action: 'You can now opt-in to use webpack 5 by using Yarn and adding `"resolutions": {"webpack": "^5.0.0"}` to your `package.json`.' },
```

Comment:
Not sure if we should recommend this just yet. `ng serve` is buggy and we're still verifying which webpack plugins we depend on function with v5.